.github/dependabot.yml

```diff
@@ -5,10 +5,10 @@ updates:
     schedule:
       interval: weekly
       day: tuesday
-    open-pull-requests-limit: 99
+    open-pull-requests-limit: 2
```

Comment:
Why this value got reduced?

Reply:
99 seemed a bit wild to me, I'd prefer to avoid having a wall of PRs from dependabot like it is now with `oslo`
it obscures the important stuff, which are user PRs

Reply:
On the other hand 2 feels a bit too small but we can live with that and see whether it makes any problems